cl/settings/10-public.py

```diff
@@ -276,10 +277,13 @@
 )
 # This is where things get collected to
 STATIC_ROOT = os.path.join(INSTALL_ROOT, "cl/assets/static/")
+if not any([TESTING, DEBUG]):
+    STATICFILES_STORAGE = "cl.lib.storage.SubDirectoryS3ManifestStaticStorage"
 
 # Where should the bulk data be stored?
 BULK_DATA_DIR = os.path.join(INSTALL_ROOT, "cl/assets/media/bulk-data/")
 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
This looks like it was added by Nick. It's a bit weird because we already have `BASE_DIR` configured in settings.py. Can you try removing it and make sure everything still works? I'm not sure what this variable is supposed to do, honestly, but it's not one that we usually would need to tinker with.

Reply:
The only use of this would have been to point the webpack loader to the correct directory. So if that is sorted out, you could probably delete with no effect on anything else.

Reply:
I took it out and dont see any issues building from Webpack.